### PR TITLE
Update PAR expiration till 2023

### PR DESCRIPTION
**Pull request by shujchen-oracle**

### CODE REVIEW

File: ai-decision-services/ai-ad-dpp-engine/04-prepare-config-data-and-app/prepare-config-data-and-app.md [hunk 1]
```diff
@@ -216,8 +217,9 @@ Driver configuration files used in this workshop are provided below.
 4. Populate compartmentId with compartment OCID from **Lab 2**.
 
 ## 3. Upload Driver Configuration Files
-*   Upload OCI AD model training configuration file into **training-config-bucket** in OCI Object Storage Bucket, created in **Lab 4**
-*   Upload OCI AD inferencing configuration file into **inferencing-config-bucket**
+*   Upload OCI AD model training configuration file into **training-config-bucket** in OCI Object Storage Bucket, created in **Lab 4**.
+*   Upload OCI AD inferencing configuration file into **inferencing-config-bucket**.
```
Comment (bug-catcher) [inline ai-decision-services/ai-ad-dpp-engine/04-prepare-config-data-and-app/prepare-config-data-and-app.md:221]: pls revert, modelId is not required

File: ai-decision-services/ai-ad-dpp-engine/08-run-tests/run-tests.md [hunk 2]
```diff
@@ -59,6 +59,8 @@ Details of the trained model such as FAP(False Alarm Probability) can be checked
 
 The application assumes the user has succesfully performed model training using the pipeline prior to inference.
 
+Since your training is done, you should see your trained model under **Analytics & AI** -> **AI Service** -> **Anomaly Detection** -> \<generated-project\> -> **aianomalydetectionmodel_\<timestamp\>** -> OCID. Copy the OCID and put that under `modelId` inside your `inference-config.json`.
```
Comment (bug-catcher) [inline ai-decision-services/ai-ad-dpp-engine/08-run-tests/run-tests.md:62]: pls revert, modelId is not required

File: ai-decision-services/ai-ad-dpp-engine/04-prepare-config-data-and-app/prepare-config-data-and-app.md (hunk 1 above)
Comment (shujchen-oracle) [reply ai-decision-services/ai-ad-dpp-engine/04-prepare-config-data-and-app/prepare-config-data-and-app.md:220]: > pls revert, modelId is not required

Good catch

**Review by bug-catcher — APPROVED** [review]

**Review by ganrad — APPROVED** [review]
Looks good.  Thanks.

**Pull request merged**